src/Altinn.Platform/Altinn.Platform.Storage/Storage/Helpers/InstanceHelper.cs

```diff
@@ -63,6 +63,59 @@ public static List<MessageBoxInstance> ConvertToMessageBoxInstance(List<Instance
             return messageBoxInstances;
         }
 
+        /// <summary>
+        /// Converts to a simpler instance object that includes some application metadata
+        /// </summary>
+        public static MessageBoxInstance ConvertToMessageBoxInstance(Instance instance)
+        {
+            InstanceStatus status = instance.Status ?? new InstanceStatus();
+            DateTime? visibleAfter = instance.VisibleAfter;
+
+            string instanceId = instance.Id.Contains("/") ? instance.Id.Split("/")[1] : instance.Id;
```

Comment:
Using instanceGuid here might avoid confusion.. as the are only using the last part of what is defined as instanceId in Storage?